src/Engine/System/SoundSystem.cpp

```diff
@@ -0,0 +1,80 @@
+#include "SoundSystem.hpp"
+
+const Vector<SoundComponent *> & SoundSystem::s_soundComponents(Scene::getComponents<SoundComponent>());
+String SoundSystem::SOUND_DIR = "../resources/soundeffects/";
+Vector<String> SoundSystem::soundfiles = {
```

Comment:
TODO : have these loaded through json or manually in main

Reply:
I don't know how to use json. In my newest version, they're automatically loaded based on a text file containing all sound effect file names. The text file is updated by a python script.